doc/install/install.rst

```diff
@@ -1,296 +1,147 @@
 .. index:: Installing QIIME
 
-=========================
-QIIME installation guide
-=========================
+========================
+QIIME Installation Guide
+========================
 
-QIIME consists of native Python 2 code and additionally wraps many external applications. This gives the user flexibility to easily build their own analysis pipelines, making use of popular microbial community analysis tools. QIIME handles the processing of input and output of these applications, so the user can spend time analyzing their data rather than parsing, writing, and converting file formats.
+Installing Conda and QIIME on Mac OS X and Linux
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-As a consequence of this *pipeline* architecture, **QIIME has a lot of dependencies and can (but doesn't have to) be very challenging to install**.
+Step 1: Install the `Miniconda <http://conda.pydata.org/miniconda.html>`__ version of Conda.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We therefore break this guide down into several parts:
+Conda is a Python distribution, package manager, and virtual environment
+solution. While QIIME 1 is Python 2 software, we recommend installing
+Miniconda with Python 3 (miniconda3), as many bioinformatics packages
+are now transitioning to Python 3. You can still install Python 2
+software with miniconda3 by passing the python=2.7 flag when you create
+a new environment; otherwise the default Python version will be Python
+3.
 
- - First we tell you how you can `avoid installing QIIME natively`__, and instead use pre-built virtual machines or MacQIIME.
+You should next work through the tutorials on the Miniconda install page
+linked above. We recommend setting up a QIIME 1.9.1 environment for your
+Miniconda installation, so pay particular attention to the virtual
+environment management in this tutorial. This will eventually allow you
+to install QIIME 2 alongside QIIME 1 on your system. It will also
+facilitate easy installation of the many other bioinformatics tools
+supported by `bioconda <https://bioconda.github.io/>`__.
 
- __ vm-or-macqiime_
+Step 2: Create your qiime191 environment and install QIIME.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
- - Then we show how to `install QIIME natively with a minimal (base) install.`__ This is easy, and is sufficient for the vast majority of users.
+::
 
-__ native-base_
+    conda config --add channels bioconda
+    conda create -n qiime191 python=2.7 qiime matplotlib=1.4.3 mock nose
```

Comment:
Do we want to suggest additional packages to include in the environment that might be useful? eg Pandas, Seaborn, Jupyter, Emperor, others?